gitpod-io/gitpod
# Make JetBrains stable GHA use the minimal major build version

## Description


## Related Issue(s)

Fixes EXP-1836

## How to test

- Open this PR with Gitpod
- Unit tests should pass
- Script should only update `workspace.yaml` file, but not gradle stable file
```
cd components/ide/jetbrains/image/gha-update-image
yarn && npm i -g bun
bun test
bun run index.ts
```

## Documentation


#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

</details>

<details>
<summary>Installer</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Saves cost. Untick this only if you're really sure you need a non-preemtible machine.
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
</details>

/hold
